test/browser/render.js

```diff
@@ -472,8 +472,8 @@ describe('render()', () => {
 	it('should reorder child pairs', () => {
 		let root = render((
 			<div>
-				<a>a</a>
-				<b>b</b>
+				<a key="a">a</a>
```

Comment:
Maybe there ought to be a test without the keys to verify that reordering still works, just not with strict equality of the last two assertions (since without the keys, the resulting nodes will be correct, just not reused).